### PR TITLE
3193 update ubla

### PR DESCRIPTION
**Pull request by amitchauhan1007**

### CODE REVIEW

File: google/cloud/storage/bucket_metadata.h [hunk 1]
```diff
@@ -142,11 +142,11 @@ std::ostream& operator<<(std::ostream& os, CorsEntry const& rhs);
  *
  * @see Before enabling Uniform Bucket Level Access please
  *     review the [feature documentation][bpo-link], as well as
- *     ["Should you use Bucket Policy Only?"][bpo-should-link].
+ *     ["Should you use Uniform bucket-level access ?"][bpo-should-link].
  *
- * [bpo-link]: https://cloud.google.com/storage/docs/bucket-policy-only
+ * [bpo-link]: https://cloud.google.com/storage/docs/uniform-bucket-level-access
```
Comment (manish-qlogic) [inline google/cloud/storage/bucket_metadata.h]: `bpo-link` can be changed to `ubla-link`. 

Please check name on other places of this PR.

File: google/cloud/storage/bucket_metadata.h [hunk 2]
```diff
@@ -142,11 +142,11 @@ std::ostream& operator<<(std::ostream& os, CorsEntry const& rhs);
...
  * [bpo-should-link]:
- * https://cloud.google.com/storage/docs/bucket-policy-only#should-you-use
+ * https://cloud.google.com/storage/docs/uniform-bucket-level-access#should-you-use
```
Comment (manish-qlogic) [inline google/cloud/storage/bucket_metadata.h]: extra space at the end will cause build to fail, 

use `camke-format` to format the code before commit.

**Review by manish-qlogic — CHANGES_REQUESTED** [review]

**Comment by amitchauhan1007** [comment]
changes are done. please review

File: google/cloud/storage/client.h [hunk 3]
```diff
@@ -333,13 +333,14 @@ class Client {
...
-   *     [feature documentation][bpo-link], as well as
-   *     ["Should you use Bucket Policy Only?"][bpo-should-link].
+   * @see Before enabling Uniform Bucket Level Access please review the
+   *     [feature documentation][ubla-link], as well as
+   *     ["Should you use Uniform bucket-level access ?"][ubla-should-link].
    *
-   * [bpo-link]: https://cloud.google.com/storage/docs/bucket-policy-only
-   * [bpo-should-link]:
-   * https://cloud.google.com/storage/docs/bucket-policy-only#should-you-use
+   * [ubla-link]:
```
Comment (manish-qlogic) [inline google/cloud/storage/client.h:340]: the link appears in the next line. I think the generated document will break due to this. We must have the link on the same line. 
Please confirm this by looking at generated documents.

Comment (amitchauhan1007) [reply google/cloud/storage/client.h:340]: Hi Manish
When the links are moved to the next line, docker build throws complains about formatting. Please refer attached screenshot image.
docker_build_3193

And when the links are on the same line, as already, docker build doesn't through any formatting complain.

Please suggest.

**Pull request closed without merging**